NGitLab/Models/JobCommon.cs

```diff
@@ -0,0 +1,94 @@
+﻿using System;
+using System.Runtime.Serialization;
+using System.Text.Json.Serialization;
+
+namespace NGitLab.Models
+{
+    [DataContract]
+    public class JobCommon
```

Comment:
We should match the GitLab naming (https://gitlab.com/gitlab-org/gitlab/-/blob/master/lib/api/entities/ci/job_basic.rb)

```suggestion
    public class JobBasic
```